Assets/MixedRealityToolkit-Tests/Core/TestFixture_01_MixedRealityToolkitTests.cs

```diff
@@ -0,0 +1,543 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Core.Interfaces;
+using Microsoft.MixedReality.Toolkit.Core.Services;
+using Microsoft.MixedReality.Toolkit.Tests.Services;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Microsoft.MixedReality.Toolkit.Tests.Core
+{
+    public class TestFixture_01_MixedRealityToolkitTests
+    {
+        #region Service Locator Tests
+
+        [Test]
+        public void Test_01_InitializeMixedRealityToolkit()
+        {
+            TestUtilities.CleanupScene();
+
+            TestUtilities.InitializeMixedRealityToolkit();
+            GameObject gameObject = GameObject.Find(nameof(MixedRealityToolkit));
+            Assert.AreEqual(nameof(MixedRealityToolkit), gameObject.name);
+        }
+
+        [Test]
+        public void Test_02_TestNoMixedRealityConfigurationFound()
+        {
+            LogAssert.Expect(LogType.Error, "No Mixed Reality Configuration Profile found, cannot initialize the Mixed Reality Toolkit");
+
+            MixedRealityToolkit.Instance.ActiveProfile = null;
+
+            Assert.IsFalse(MixedRealityToolkit.HasActiveProfile);
+            Assert.IsNull(MixedRealityToolkit.Instance.ActiveProfile);
+
+            TestUtilities.CleanupScene();
+        }
+
+        [Test]
+        public void Test_03_CreateMixedRealityToolkit()
+        {
+            TestUtilities.InitializeMixedRealityToolkitScene();
+
+            // Create Test Configuration
```

Comment:
This comment does not appear accurate, the line above appears to do the configuration creation and the asserts validate the results.